.circleci/cci_create_release_and_snapshot.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+REPOSITORY=https://github.com/gresham-computing/openid-connect-server
+MASTER_BRANCH=1.3.x
+
+function get_version {
+    local currentVersion=$(mvn -Dexec.executable='echo' -Dexec.args='${project.version}' --non-recursive exec:exec -q)
+    IFS='-' read -r -a parts <<< "$currentVersion"
+
+    local NUMBER="$((${parts[1]} + 1))"
```

Comment:
OCD me, the variable could ideally say 'next' number, given it has a sneaky +1 on the end.